src/documentation/forgot.doc.js

```diff
@@ -0,0 +1,50 @@
+import responses from './response';
+
+
+const forgot = {
+    '/users/forgot': {
```

Comment:
change route to " /users/forgot-password", and implement the same on reset password route